src/main/kotlin/com/sourcegraph/cody/chat/ChatSession.kt

```diff
@@ -122,8 +126,6 @@ private constructor(private val project: Project, newSessionId: CompletableFutur
     }
 
     try {
-      GraphQlLogger.logCodyEvent(project, "chat-question", "submitted")
```

Comment:
so this was the place causing the troubles?

Reply:
Yes